ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-tags-input/README.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Components / Form Tags Input / README" />
+
+# Form Tags Input
+
+Documentation and examples for Form Tags Input, a component to specify a list of values based on material chip-list
+
+## Demo
+
+Component inside a Form field
+
+<Story id="components-form-tags-input--with-initial-value"/>
+
+Component without a Form field
+
+<Story id="components-form-tags-input--without-form-field"/>
+
+
+
+## Usage
+
+**Inputs**
+ - `ariaLabel`: `string` - Aria label for the input
+ - `addOnBlur`: `boolean` - Whether or not the chipEnd event will be emitted when the input is blurred.
+ - `tagValidationHook`: `function(tag, validationCallback())` - Function called each time a tag is added. Useful to hook inside the addition process to, for instance, have custom validation in the components using `gio-form-tags-input`.
+ - `placeholder`: `string` - The input's placeholder text.
+ - `required`: `boolean` - Whether the input requires to have at least one value.
+ - `disabled`: `boolean` - Whether the input is disabled.
+
+Example:
+
+```html
+<mat-form-field appearance="fill" style="width:100%">
+  <mat-label>Tags</mat-label>
+  <gio-form-tags-input
+    [disabled]="disabled"
+    [required]="required"
+    [placeholder]="Add a tag"
+    [ngModel]="tags"
```

Comment:
What do you think about preferring an example with Reactive Forme, to promote the good practice of using it in these conditions. or to say that it works with both 🤔 

```
 [formControl]="tagsControl"
```

Reply:
what about this one : 

```
    <mat-form-field appearance="fill">
      <mat-label>My tags</mat-label>
      <gio-form-tags-input
        [required]="required"
        [placeholder]="placeholder"
        [formControl]="tagsControl"
      ></gio-form-tags-input>
      <mat-error>Error</mat-error>
    </mat-form-field>
```

Reply:
I leave just the example with Reactive Form 😉